project/Dependencies.scala

```diff
@@ -115,7 +115,12 @@ object Dependencies {
   val Sqs = Seq(
     libraryDependencies ++= Seq(
       "com.amazonaws"     %  "aws-java-sdk-sqs"    % "1.11.109",             // ApacheV2
-      "org.elasticmq"     %% "elasticmq-rest-sqs"  % "0.13.4"        % Test, // ApacheV2
+      "org.elasticmq"     %% "elasticmq-rest-sqs"  % "0.13.4"        % Test excludeAll(
+        // elasticmq-rest-sqs depends on Akka 2.5, exclude it, so we can choose Akka version
+        ExclusionRule(organization = "com.typesafe.akka") //
```

Comment:
That's an interesting one. strictly speaking I think that it might work when running with 2.4, because the library is compiled with an later akka version and we are only binary **backwards** compatible.